spring-cloud-starter-tencent-polaris-router/src/test/java/com/tencent/cloud/polaris/router/beanprocessor/PolarisLoadBalancerCompositeRuleBeanPostProcessorTest.java

```diff
@@ -0,0 +1,147 @@
+package com.tencent.cloud.polaris.router.beanprocessor;
+
+import com.netflix.loadbalancer.AbstractLoadBalancerRule;
+import com.netflix.loadbalancer.BestAvailableRule;
+import com.netflix.loadbalancer.IRule;
+import com.netflix.loadbalancer.RandomRule;
+import com.netflix.loadbalancer.RoundRobinRule;
+import com.netflix.loadbalancer.ZoneAvoidanceRule;
+import com.tencent.cloud.polaris.loadbalancer.config.PolarisLoadBalancerProperties;
+import com.tencent.cloud.polaris.router.PolarisLoadBalancerCompositeRule;
+import com.tencent.cloud.polaris.router.config.RibbonConfiguration;
+import com.tencent.polaris.client.api.SDKContext;
+import com.tencent.polaris.router.api.core.RouterAPI;
+import com.tencent.polaris.router.client.api.DefaultRouterAPI;
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.springframework.boot.autoconfigure.AutoConfigurations;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+import org.springframework.cloud.netflix.ribbon.RibbonAutoConfiguration;
+import org.springframework.cloud.netflix.ribbon.RibbonClient;
+import org.springframework.cloud.netflix.ribbon.RibbonClients;
+import org.springframework.cloud.netflix.ribbon.SpringClientFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+
+/**
+ * @author derekyi
```

Comment:
Add `Test for {@link PolarisLoadBalancerCompositeRuleBeanPostProcessor}.`

Reply:
Done.